Nexus.Sdk.Shared.Tests/CustomerRequestBuilderTests.cs

```diff
@@ -443,6 +443,7 @@ public void CustomerRequestBuilderTests_Build_UpdateCustomerRequest()
         {
             var request = new UpdateCustomerRequestBuilder("MOCK_CUSTOMER")
                 .SetTrustLevel("Trusted")
+                .SetReason("ReasonCode")
```

Comment:
`SetReasonCode`

Reply:
[Update correct function on Update Customer request for ReasonCode property](https://github.com/QuantozTechnology/Nexus.Sdk.Dotnet/pull/47/commits/e7952f32cda316fdee1ca92f9992e425ae0af569)